seqr/views/apis/data_manager_api_tests.py

```diff
@@ -823,7 +893,7 @@ def _test_basic_data_loading(data, num_parsed_samples, num_loaded_samples, indiv
                         'dbEntity': 'Sample', 'updateType': 'bulk_create',
                         'entityIds': response_json['sampleGuids'] if num_created_samples > 1 else
                             [response_json['sampleGuids'][-1]],
-                    }})] + (additional_logs or [])
+                    }})] + (additional_logs or []) if num_created_samples else additional_logs or []
```

Comment:
the conditional belongs above when you create `additional_logs`, there is no reason to create a variable on the above line and then immediately follow it with a conditional where it is not used